my_game.py

```diff
@@ -316,11 +318,9 @@ def setup(self):
         )
         
         # Temporary Asteroid test
-        new_asteroid = Asteroid(
-            center_x=SCREEN_WIDTH//2,
-            center_y=SCREEN_HEIGHT//2
-        )
-        self.asteroid_list.append(new_asteroid)
+        for r in range(5):
+            new_asteroid = Asteroid()
+            self.asteroid_list.append(new_asteroid)
```

Comment:
No need fot the variable new_asteroid. Let's create the Asteroid() directly in the .append()

Reply:
Oh alright, I understand